packages/site-admin/.gitignore

```diff
@@ -0,0 +1 @@
+dist
```

Comment:
Can you add `package.tgz`? Or would that belong in the repo level gitignore?

Reply:
Not sure, to be honest. Ignoring the file could confuse when the new package is created. How do you realize there is a newly built package? 
I don't have a strong opinion about ignoring the package, either. It's just that I haven't seen that in other calypso packages.

Reply:
Sure, sounds good to follow precedent. We may never `pack` as part of the development story in any case

Reply:
Exactly. I used it as a easy way to test the building system of the package works. It feels a bit trivial.